src/SimplygonFunctionApp/AzRemeshFn.cs

```diff
@@ -1,195 +1,390 @@
-                    sgScene.GetMaterialTable().Clear();
-
-                }
-                using (Simplygon.spSceneExporter sgSceneExporter = sg.CreateSceneExporter())
-                {
-                    sgSceneExporter.SetScene(sgScene);
-                    sgSceneExporter.SetExportFilePath(filePathOutput);
-
-                    if (!sgSceneExporter.RunExport())
-                    {
-                        log.LogError($"Failed to save RemeshingOutput {filePathOutput}");
-                        throw new Exception("Failed to save RemeshingOutput.");
-                    }
-                    return filePathOutput;
-                }
-            }
-        }
-
-        private static async Task<PostData> ProcessParameters(HttpRequest req)
-        {
-            PostData data = new PostData();
-
-            if (req.Method == "GET")
-            {
-                var inputZip = req.Query["InputZipUri"];
-                if (!string.IsNullOrEmpty(inputZip))
-                {
-                    data.InputZipUri = new Uri(inputZip);
-                    var _onScreenSize = req.Query["OnScreenSize"];
-                    uint onScreenSize = 0;
-                    data.OnScreenSize = DEFAULT_SCREEN_SIZE;
-                    if (!string.IsNullOrEmpty(_onScreenSize) && uint.TryParse(_onScreenSize, out onScreenSize) && onScreenSize > MIN_ONSCREEN_SIZE)
-                    {
-                        data.OnScreenSize = onScreenSize;
-                    }
-                    else
-                    {
-                        data.OnScreenSize = DEFAULT_SCREEN_SIZE;
-                    }
-                }
-            }
-            else if (req.Method == "POST")
-            {
-                var content = await new StreamReader(req.Body).ReadToEndAsync();
-                data = JsonConvert.DeserializeObject<PostData>(content);
-            }
-
-            return data;
-        }
-
-        private static string ToOutputPath(DirectoryInfo outputDir, string inputFilePath, string suffix = ".remesh.")
-        {
-            var newFileName = Path.ChangeExtension(inputFilePath, $"{suffix}{Path.GetExtension(inputFilePath)}");
-            return Path.Combine(outputDir.FullName, Path.GetFileName(newFileName));
-        }
-    }
-}
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using System.Net.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.IO.Compression;
+using System.Linq;
+using System.Diagnostics;
+using SimplygonFunctionApp.Extensions;
+using SimplygonFunctionApp.Models;
+
+/// <summary>
+/// IMPORTANT: THIS IS A PROOF-OF-CONCEPT, NOT INTENTED FOR PRODUCTION USE
+/// </summary>
+namespace SimplygonFunctionApp
+{
+    public static class AzRemeshFn
+    {
+        public const uint DEFAULT_SCREEN_SIZE = 300;

```

Comment:
Good practice for contant naming: use the same as for fonctions. in this case it will be: DefaultScreenSize